llvm-bazel/llvm-project-overlay/llvm/BUILD

```diff
@@ -30,2213 +14,1351 @@ exports_files(["LICENSE.TXT"])
 
 llvm_host_triple = "x86_64-unknown-linux_gnu"
 
+# FIXME: Would be nice to have a select here on the compiler so that we can use
+# a more minimal list for Clang than we need for GCC.
+llvm_copts = [
+    "$(STACK_FRAME_UNLIMITED)",
+    "-Wno-unused-parameter",
+    "-Wno-comment",
+    "-Wno-class-memaccess",
+    "-Wno-maybe-uninitialized",
+    "-Wno-misleading-indentation",
```

Comment:
I see them every build step unto infinity :-D What about moving these to the bazelrc then where it can be compiler dependent? (Or just fixing them upstream works :grin:)

Reply:
What we do in IREE is use per_file_copt to [disable diagnostics on code we don't control and set `-Werror` on code we do](https://github.com/google/iree/blob/dcef2858374139821a7dbe2f5177706211438c71/build_tools/bazel/iree.bazelrc#L67-L70).

Reply:
I'd like for the flags needed for LLVM to be provided by the LLVM BUILD file though... We can give up at some point, but I'm happy to keep iterating here for now, as long as the current state works for IREE?

Reply:
Yeah agreed. And this should work fine for IREE. Thanks :-)